sound/soc/intel/boards/sof_sdw.c

```diff
@@ -256,6 +256,16 @@ static const struct dmi_system_id sof_sdw_quirk_table[] = {
 					SOF_BT_OFFLOAD_SSP(2) |
 					SOF_SSP_BT_OFFLOAD_PRESENT),
 	},
+	{
+		.callback = sof_sdw_quirk_cb,
+		.matches = {
+			DMI_MATCH(DMI_SYS_VENDOR, "Dell Inc"),
+			DMI_EXACT_MATCH(DMI_PRODUCT_SKU, "0AF3"),
+		},
+		/* No Jack */
+		.driver_data = (void *)(SOF_SDW_TGL_HDMI |
+					SOF_SDW_FOUR_SPK),
```

Comment:
commit message: when we use the term 'DMIC' we usually mean the PCH-attached DMICs, this is not the case here.

'This product supports SoundWire capture from local microphones and two SoundWire amplifiers"

Reply:
Thanks for your suggestion, I have changed all the commit messages.